docs/pages/identity-governance/access-requests/automatic-reviews.mdx

```diff
@@ -90,36 +71,90 @@ does not support wildcard or regular expressions.
   by the IdP.
 </Admonition>
 
-## Step 3/4. Create automatic review rule
+## Step 3/5. Ensure resource labels
+
+List your SSH servers to identify available labels:
+
+```code
+$ tsh ls
+Node Name   Address    Labels
+----------- ---------- -------------------------------
+teleport-00 ⟵ Tunnel   env=demo
+```
+
+This example resource has the label `env: demo`. If your resource has not been
+labeled, reconfigure your resource with a label. Alternatively, you can add a
+label using a `server_info` resource.
+
+1. Retrieve information about the SSH server.
+
+```code
+$ tctl get node/teleport-00
+kind: node
+metadata:
+  expires: "2025-08-14T18:49:19Z"
+  name: 116b08d2-7167-4eab-85a8-cf93f054b217
+spec:
+  addr: ""
+  hostname: teleport-00
+```
+
+2. Create the corresponding `server_info.yaml` for the node above.
+
+```yaml
+# server_info.yaml
+kind: server_info
+metadata:
+  # Must match the `metadata.name` field of the node, prefixed with `si-`.
+  name: si-116b08d2-7167-4eab-85a8-cf93f054b217
+spec:
+  new_labels:
+    env: demo
+```
+
+3. Create and verify the `server_info` resource. It may take several minutes for
+the new labels to appear. Note that labels added this way will be prefixed with
+`dynamic/`.
+
+```code
+# Create the `server_info` resource
+$ tctl create server_info.yaml
+
+# Verify that the labels have been applied
+$ tsh ls
+Node Name   Address    Labels
+----------- ---------- -------------------------------
+teleport-00 ⟵ Tunnel  dynamic/env=demo
```

Comment:
For most cases, we can keep it simple: ask users to add a label to the resource, e.g., env: "foobar" or use exist resource with a lable. 
For teams that need advanced strategies, we can point them to the Teleport docs on labels: https://goteleport.com/docs/zero-trust-access/management/admin/labels/


My point is that this guide cover in 60% the `Ensure resource labels` where the essence `Configure Automatic Reviews` is only 40% of this docs.

Reply:
💯

Reply:
That's a good point. I've simplified this step and just added a note in the `Next Steps` section to follow that guide for more details about resource label configuration.